packages/examples/src/bakeoff/reddit-qa/ai-jsx.tsx

```diff
@@ -1,47 +1,54 @@
 // This script assumes that ./load-articles has been run first.
+import * as LLMx from 'ai-jsx';
+import { Corpus, CorpusLoading, DocsQA, Document, LocalCorpus, defaultChunker } from 'ai-jsx/batteries/docs';
+import { showInspector } from 'ai-jsx/core/inspector';
 import { globbySync } from 'globby';
+import { loadJsonFile } from 'load-json-file';
 import path from 'node:path';
 import { fileURLToPath } from 'node:url';
-import { loadJsonFile } from 'load-json-file';
-import * as LLMx from 'ai-jsx';
-import { DocsQA, defaultChunkMany, DefaultInMemoryVectorStore } from 'ai-jsx/batteries/docs';
-import { OpenAIEmbeddings } from 'langchain/embeddings/openai';
+import { JsonObject } from 'type-fest';
 import { Article } from './load-articles.mjs';
-import _ from 'lodash';
-import { showInspector } from 'ai-jsx/core/inspector';
 
 const dirname = path.dirname(fileURLToPath(import.meta.url));
 
+interface RedditMetadata extends JsonObject {
```

Comment:
I don't think we want `extends JsonObject` here unless you want to let `RedditMetadata` have arbitrary other properties.

Reply:
I added this to satisfy the constraint that RedditMetadata is a Jsonifiable.  Is there a better way to do this?

Reply:
Yep! 

https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbwFIGcIDsDyAjAVgUwGMYBfOAMyghDgHIYBPMfAWnPxRloG4AoX4Ohj4o5AIaF8cAEpx8AD2HoAJijioMOAsUS84FCBABccbIYA2+Mel4l+youbFQphDJzhQT0vrwD0fnAA6lIA7tbwMAAWwGowEHJQVFAANKYArvDA8MoQHHDoEDBp2ERi6ShSAAYKSqrqaFh4RDBVcCBW6HFRIq7WplLO2NlQzgxwANb4DCgAdLxQs2IoyuS+jMxwALIVMABC+BpN2jAAPAAqAHxwALxw53KK+CpqR1otcAD893Am6PgANxE6yYUmkACZbttdgc3s1iKcEHoDMZTBZOrZLvZHM5XO54FBwd5wb4AvcYmoRMkUGllnBQvhaMo4FEIMx5oSlis1vwydIAMz04DmcxwNwdOAQTJweIDGWguBVf5AqBtazM6IVOXpdAVMTYSwlMoVKTZChiYXdVw9QgTeYbMGCu47TiwxrvBFI-TkQwmMwQSzWFLI8joEwACgAlLdrgg7CQsbwgA